client/src/scss/_settings.scss

```diff
@@ -71,6 +71,9 @@ $global-text-direction: ltr;
 $porcelain: #F7F8F9;
 $mercury: #E1E1E1;
 $scorpion: #5e5e5e;
+$paw-red: #FF6356;
+$paw-yellow: #FFDF00;
+$paw-green: #77F68F;
```

Comment:
These three colors are defined similarly as `$success-color`, `$warning-color`, and `$alert-color`. I think we should do one of 2 things:

* Move the `paw-*` colors above the Foundation settings colors, and assign the Foundation colors to be the same as paw's, E.g. 
```
$paw-red: #FF6356;
$alert-color: $paw-red;
```
* Replace the `paw-*` colors with Foundation's and change the Foundation colors to the colors you defined here.